src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/styles/blocks/variables';
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-family: $main-font-family;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-size: 100% 100%;
```

Comment:
You have two `background-size` properties in this rule. The second one will overwrite the first one. Please remove the unnecessary property.